internal/client/client.go

```diff
@@ -100,30 +105,33 @@ func Run(addr, fingerprint, proxyAddr string) {
 
 	sshPriv, sysinfoError := keys.GetPrivateKey()
 	if sysinfoError != nil {
-		log.Fatal("Getting private key failed: ", sysinfoError)
+		log.Println("Getting private key failed: ", sysinfoError)
```

Comment:
Clients still need to generate a private key, and if they cant that is a fatal error.

Reply:
Once username and password authentication is supported, this key is no longer a key error